.travis.yml

```diff
@@ -18,6 +18,9 @@ script:
   - bundle exec rspec spec
   - for f in examples/*.sh.txt; do echo "checking bash syntax $f"; bash -n $f; done
 
+after_success:
+- codeclimate-test-reporter
```

Comment:
https://travis-ci.org/travis-ci/travis-build/jobs/175423342#L912 it's not there. Do we need it?